books/architecture/src/resources/cap/threads.md

```diff
@@ -0,0 +1,2 @@
+# Index of threads
+This is an index of all of the system threads Cuprate actively uses.
```

Comment:
Currently, is this just `global rayon` + `global tokio` + `blockchain` + `txpool`?

@Boog900 are there any threads (ephemeral/persistent) spawned in any of your code? Or any code that we call out to?

Reply:
We are going to have a thread that listens to Stdin and handles commands. Also the `blockchain` + `txpool` will use the same threadpool